libs/generic-view/ui/src/lib/data-rows/text-formatted.tsx

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import React from "react"
+import styled from "styled-components"
+import { APIFC, withData } from "generic-view/utils"
+import Markdown from "react-markdown"
+
+interface Data {
+  content: string
+}
+
+const TextFormatted: APIFC<Data> = ({ data }) => {
+  return <Content>{data?.content}</Content>
+}
+
+export default withData(TextFormatted)
+
+const Content = styled(Markdown)`
```

Comment:
Below styles will be improved in future, but we need to get final designs first (in progress).